src/main/java/Task.java

```diff
@@ -0,0 +1,22 @@
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
```

Comment:
Avoid unnecessary use of this with fields.
Refer to: https://se-education.org/guides/conventions/java/index.html#variables